STYLEGUIDE.md

```diff
@@ -0,0 +1,83 @@
+
+Thanks for contributing to Lem! Here are some guidelines that we follow in the codebase.
+
+In general, we follow the Google guide: https://google.github.io/styleguide/lispguide.xml
+
+
+## Alexandria and other util libraries
+
+Lem depends on `alexandria`, so you can use `if-let`, `when-let` and similar functions.
+
+You can `:import-from` these symbols in your package definition, use the `alexandria:` package prefix, and define a package-local nickname.
+
+Try to not use new utility functions that you don't see in the codebase yet.
+
+For instance, we won't use `alexandria-2:line-up-first`.
+
+Try to not use `alexandria:curry` and prefer higher-order functions.
+
```

Comment:
How about also specifying things like avoiding dynamic writing (e.g. uiop:symbol-call) and not ignoring compiler warnings?